### ISSUE TITLE
Load default avatar image for a user with no avatar

### ISSUE THREAD

**Issue by ennukee** [ennukee/aniupdater#28]
Currently the only way to log out is via the avatar in the top right when logged in.

This is not present when users do not have an avatar on AniList, meaning they cannot log out without removing the token themselves or clearing browser cache.
